ttls/cli.py

```diff
@@ -113,13 +113,14 @@ async def command_movie(t: Twinkly, args: argparse.Namespace):
 
 async def command_static(t: Twinkly, args: argparse.Namespace):
     await t.interview()
-    m = re.match(r"(\d+),(\d+),(\d+)", args.colour)
-    if m is not None:
-        rgb = (int(m.group(1)), int(m.group(2)), int(m.group(3)))
+    if m := re.match(r"(\d+),(\d+),(\d+),(\d+)", args.colour):
+        rgbw = (int(m.group(1)), int(m.group(2)), int(m.group(3)), int(m.group(4)))
+    elif m := re.match(r"(\d+),(\d+),(\d+)", args.colour):
+        rgbw = (int(m.group(1)), int(m.group(2)), int(m.group(3)), None)
     else:
         c = TwinklyColour(args.colour)
-        rgb = (int(c.red * 255), int(c.green * 255), int(c.blue * 255))
-    return await t.set_static_colour(rgb)
+        rgbw = (int(c.red * 255), int(c.green * 255), int(c.blue * 255), None)
+    return await t.set_static_colour(rgbw)
```

Comment:
It would be easier to read if `set_static_colour` is called with a `TwinklyColour` instead of a tuple.